src/routes/dashboard/components/Dashboard.js

```diff
@@ -87,24 +87,29 @@ class Dashboard extends React.Component<PropsType> {
         path: 'events',
         thumbnail: eventsIcon,
         isExternalUrl: false,
-        onTilePress: () =>
-          navigateToEvent({ cityCode, language, path: null }),
+        onTilePress: () => navigateToEvent({
+          cityCode,
+          language,
+          path: null
+        }),
         notifications: 0
-      }),
-      isNewsEnabled && new TileModel({
+      }))
+    }
+
+    if (isNewsEnabled) {
+      tiles.push(new TileModel({
         title: t('news'),
         path: 'news',
         thumbnail: newsIcon,
         isExternalUrl: false,
         onTilePress: () =>
-          navigateToNews({ cityCode, language, path: null, type: 'local' }),
-        notifications: 0
+          navigateToNews({ cityCode, language, newsId: null, type: LOCAL })
```

Comment:
What happens if only tunews is enabled?

Reply:
Actually this is fixed in New component mounting. But should also be fixed here.

Reply:
Sorry, I don't really understand the comment :see_no_evil: